### ISSUE TITLE
Scaling intrinsic matrix

### ISSUE THREAD

**Issue by hkashyap** [ClementPinard/SfmLearner-Pytorch#63]
Can you please explain why only the focal length is scaled, and principal point is not?

https://github.com/ClementPinard/SfmLearner-Pytorch/blob/94b261942a618cfbfa612a91c46c1922adbb3320/loss_functions.py#L21

**Comment by ClementPinard** [comment]
If by principal point you mean optical center, it is scaled. The 0:2 indicates the first two lines of the matrix.

The only value that is not scaled is the last line which is just `[0,0,1]`

**Comment by hkashyap** [comment]
Thanks, did not realize that intrinsics are from dataloader and first dim is batchsize.
